benches/benchmarks/graphs.rs

```diff
@@ -0,0 +1,116 @@
+use std::collections::HashMap;
+use criterion::{criterion_group, Criterion};
+use oasysdb::db::database::*;
+use crate::benchmarks::utils;
+
+
+fn create_database(path: String, dimension: usize) -> Database {
+    // remove the database if it already exists
+    let _ = std::fs::remove_dir_all(&path);
+    let config = Config { path, dimension };
+    let db = Database::new(config);
+    db
+}
+
+fn add_values(db: &Database, query: &Vec<Embedding>) {
+    // Add the values to the database
+    for i in 0..query.len() {
+        let embedding: Embedding = query[i].clone();
+        let value = Value { embedding, data: HashMap::new() };
+        db.set_value(&i.to_string(), value).unwrap();
+    }
+}
+
+fn build_graph(db: &Database, name: &str, ef_construction: usize, ef_search: usize) {
+    // Build the graph
+    let config = GraphConfig { name: name.to_string(), ef_construction, ef_search, filter: None };
+    db.create_graph(config).unwrap();
+}
+
+fn search_graph(db: &Database, name: &str, query: &Embedding, k: usize) {
+    // Search the graph
+    let _ = db.query_graph(name, query.clone(), k).unwrap();
+}
+
+
+fn bench_add_values(c: &mut Criterion) {
```

Comment:
Can we move this benchmark function to its own module dedicated for values operation: `benchmarks/values.rs`.